src/Apps/Gene/Components/GeneArtworkFilter.tsx

```diff
@@ -50,7 +49,6 @@ const GeneArtworkFilter: React.FC<
       aggregations={
         sidebar?.aggregations as SharedArtworkFilterContextProps["aggregations"]
       }
-      counts={sidebar?.counts as Counts}
```

Comment:
Is this OK? From what I could tell, `ArtworkFilterContext` doesn't use `counts.followedArtists` but I'm not confident I understand everything that component is doing.

Reply:
As long as Relay + TS are ok with it - we should be fine? I guess there was some over-fetching happening with some counts that weren't being used. If the `FOLLOWED_ARTISTS` aggregation wasn't requested, the count would be always 0 anyway - https://github.com/artsy/metaphysics/blob/2fe85e11efb16037c1e04c027d6b3edbac4ae3e5/src/schema/v2/filterArtworksConnection.ts#L89-L90